api/proto/lightnode/lightnode.proto

```diff
@@ -0,0 +1,65 @@
+syntax = "proto3";
+package lightnode;
+option go_package = "github.com/Layr-Labs/eigenda/api/grpc/lightnode";
+
+service LightNode {
+	// GetChunk retrieves a specific chunk held by the light node.
+	rpc GetChunk(GetChunkRequest) returns (GetChunkReply) {}
+
+	// StreamAvailabilityStatus streams the availability status of all chunks assigned to the light node.
+	// For use by a DA node for monitoring the availability of chunks through its constellation of agent light nodes.
+	rpc StreamAvailabilityStatus(StreamAvailabilityStatusRequest) returns (stream SampledChunk) {}
+}
+
+// TODO these message types are duplicated from node.proto, perhaps they should share a common definition
+
+// A request to retrieve a specific chunk.
+message GetChunkRequest {
+	// The hash of the blob's ReducedBatchHeader defined on-chain.
+	bytes batch_header_hash = 1;
```

Comment:
same as above

Reply:
updated to support lookup via both mechanisms